### PR TITLE
a2ensite: add Indonesian translation

### PR DESCRIPTION
**Pull request by punkestu**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message-and-pr-title).
- **Version of the command being documented (if known):**

### CODE REVIEW

File: pages.id/linux/a2ensite.md [hunk 1]
```diff
@@ -0,0 +1,12 @@
+# a2ensite
+
+> Izinkan sebuah host maya Apache pada OS berbasis Debian.
```
Comment (reinhart1010) [inline pages.id/linux/a2ensite.md:3]:
```suggestion
> Izinkan sebuah host maya (virtual host) Apache pada OS berbasis Debian.
```

The technical name for this is `VirtualHost`, and adding an English translation here could help some people who aren’t familiar with standard Indonesian words such as `awakutu` for `debugging`.

**Review by reinhart1010 — CHANGES_REQUESTED** [review]
Thanks for your contribution. Here are some of my notes to review.

**Review by reinhart1010 — APPROVED** [review]
Thanks!

**Review by gutjuri — APPROVED** [review]

**Review by kbdharun — APPROVED** [review]
LGTM, Thanks for your contribution.

**Pull request merged**